automation/read_as_tag.py

```diff
@@ -0,0 +1,25 @@
+import os
+from github import Github
+
+GITHUB_REPO = "mozilla/application-services"
+github_access_token = os.getenv("GITHUB_TOKEN")
+
+def get_latest_as_version():
+    g = Github(github_access_token)
+    repo = g.get_repo(GITHUB_REPO)
+    
+    latest_tag = repo.get_tags()[0].name
```

Comment:
note to self, this should probably `get_releases`, in case a non-release tag is created